test/actor.cc

```diff
@@ -159,8 +159,8 @@ TEST(Actor, TileReturnShortcuts) {
     }
   }
 
-  ASSERT_EQ(features_with_shortcuts, 2320);
-  ASSERT_EQ(features_no_shortcuts, 2281);
+  ASSERT_EQ(features_with_shortcuts, 2317);
+  ASSERT_EQ(features_no_shortcuts, 2278);
```

Comment:
so the tile and isochrone tests changed because there's a disused route tag somewhere in the utrecht data? Or is it possible it's an unwanted side effect?

Reply:
Not only `disused`, but literally every other OSM Way that is not road (no `highway` tag and not ferry) but has any of access tags.